app/packages/acter_trigger_auto_complete/lib/src/acter_trigger_autocomplete.dart

```diff
@@ -377,11 +377,15 @@ class MultiTriggerAutocompleteState extends State<MultiTriggerAutocomplete> {
 
   @override
   void dispose() {
-    _textEditingController.removeListener(_onChangedField);
-    _textEditingController.dispose();
+    if (!mounted) {
```

Comment:
sure it should be `!mounted` and not `mounted` here? I think we only want to do those if we are still mounted, no?

Reply:
We can move the discussion [here](https://github.com/acterglobal/a3/pull/2026). Closing this PR...